src/main/target/ATOMRCF405V2/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+target_stm32f405xg(ATOMRCF405V2)
```

Comment:
Can you add SKIP_RELEASES to this target?

Reply:
Sure, where does this need to be added?

Reply:
target_stm32f405xg(ATOMRCF405V2 SKIP_RELEASES)